packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/tests/index.test.js

```diff
@@ -433,4 +440,38 @@ describe('BlocksEditor toolbar', () => {
     const blocksDropdown = screen.getByRole('combobox', { name: /Select a block/i });
     expect(within(blocksDropdown).getByText(/heading 1/i)).toBeInTheDocument();
   });
+
+  it('creates a new code block without empty lines before it when you select the option in a empty editor', async () => {
+    setup(emptyInitialValue);
```

Comment:
I don't think any other test will use emptyInitialValue so you could inline the value directly